src/main/java/Main.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
+
```

Comment:
It is great that only the necessary import statements are written and * is not used to import everything.